Konglish/ARCore/Sources/Features/CardDetector.swift

```diff
@@ -45,27 +45,34 @@ class CardDetector: ARFeatureProvider {
     private func findCardAtCenter() -> CardEntity? {
         guard let arView = arView else { return nil }
         
-        // Raycast 동작
+        // RayCast 동작이 안됨
         let centerPoint = CGPoint(x: arView.bounds.midX, y: arView.bounds.midY)
         
         // .estimatedPlane - ARKit이 추정한 평면들 (바닥, 벽, 테이블 등)
-        let results = arView.raycast(from: centerPoint, allowing: .estimatedPlane, alignment: .any)
+//        let results = arView.raycast(from: centerPoint, allowing: .estimatedPlane, alignment: .any)
         
-        for result in results {
-            // columns.3 = raycast가 평면에 맞은 3D 월드 좌표
-            let worldPosition = result.worldTransform.columns.3
-            let anchor = result.anchor
-            
-            for sceneAnchor in arView.scene.anchors {
-                if let anchorEntity = sceneAnchor as? AnchorEntity,
-                   anchorEntity.anchor?.anchorIdentifier == anchor?.identifier {
-                    
-                    for child in anchorEntity.children {
-                        if let cardEntity = child as? CardEntity {
-                            return cardEntity
-                        }
-                    }
-                }
+//        for result in results {
+//            // columns.3 = raycast가 평면에 맞은 3D 월드 좌표
+//            let worldPosition = result.worldTransform.columns.3
+//            let anchor = result.anchor
+//
+//            for sceneAnchor in arView.scene.anchors {
+//                if let anchorEntity = sceneAnchor as? AnchorEntity,
+//                   anchorEntity.anchor?.anchorIdentifier == anchor?.identifier {
+//
+//                    for child in anchorEntity.children {
+//                        if let cardEntity = child as? CardEntity {
+//                            return cardEntity
+//                        }
+//                    }
+//                }
+//            }
+//        }
+        
+        // hitTest로 대체
```

Comment:
위의 코드가 지워진다면 주석도 바뀌게 될 것 같습니다

```suggestion
        // hitTest
```